PREVIEW_CHECKLIST.md

```
@@ -0,0 +1,79 @@
+7. Monitor for failed Neon branch creation
+
+## File References
+
+- **Cleanup workflow:** `.github/workflows/cleanup-preview.yml` (✅ done)
+- **Migration workflow:** `.github/workflows/drizzle-migrate.yml` (✅ fixed)
+- **Full guide:** `PREVIEW_SETUP.md` (📖 read this)
+- **Compare with:** https://github.com/FindMalek/dukkani/tree/main/.github/workflows
+
+## Expected Result
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)